book/src/qobject/cxxqtthread.md

```diff
@@ -7,16 +7,30 @@ SPDX-License-Identifier: MIT OR Apache-2.0
 
 # CxxQtThread
 
-`CxxQtThread<T>` is used for [threading](../concepts/threading.md) with QObject#s and allows you to queue events from a background thread to occur on the Qt event loop.
+`CxxQtThread<T>` is used for easy [threading](../concepts/threading.md) with QObjects.
+The QObjects generated by CXX-Qt are neither [`Sync`](https://doc.rust-lang.org/std/marker/trait.Sync.html) nor [`Send`](https://doc.rust-lang.org/std/marker/trait.Send.html).
```

Comment:
```suggestion
The QObjects generated by CXX-Qt are neither [`Send`](https://doc.rust-lang.org/std/marker/trait.Send.html) nor [`Sync`](https://doc.rust-lang.org/std/marker/trait.Sync.html).
```
match the structure of the following sentence

Reply:
done